setup.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes, but if you uninstall tox and install it again on Python 3, this problem will go away.